ui/pages/AnvilWorkspace/components/LoadWorkspaceDataForm.jsx

```diff
@@ -104,13 +104,23 @@ const createProjectFromWorkspace = postWorkspaceValues(
   window.location.href = `/project/${responseJson.projectGuid}/project_page`
 })
 
-const FORM_WIZARD_PAGES = [
-  { fields: [AGREE_CHECKBOX], onPageSubmit: postWorkspaceValues('grant_access') },
-  { fields: [DATA_BUCK_FIELD, SAMPLE_TYPE_FIELD, REQUIRED_GENOME_FIELD], onPageSubmit: postWorkspaceValues('validate_vcf') },
-  { fields: [PROJECT_DESC_FIELD, UPLOAD_PEDIGREE_FIELD] },
-]
-
-const LoadWorkspaceDataForm = React.memo(({ params }) => (
+const addDataFromWorkspace = projectGuid => values => (
+  new HttpRequestHelper(`/api/project/${projectGuid}/add_workspace_data`, (responseJson) => {
+    if (responseJson.success) {
+      window.location.href = `/project/${projectGuid}/project_page`
+    }
+  }).post({ ...values, uploadedFileId: values.uploadedFile?.uploadedFileId }))
+
+const GRANT_ACCESS_PAGE = { fields: [AGREE_CHECKBOX], onPageSubmit: postWorkspaceValues('grant_access') }
+const VALIDATE_VCF_PAGE = { fields: [DATA_BUCK_FIELD, SAMPLE_TYPE_FIELD, REQUIRED_GENOME_FIELD], onPageSubmit: postWorkspaceValues('validate_vcf') }
+const ADD_DATA_VALIDATE_VCF_PAGE = { ...VALIDATE_VCF_PAGE, fields: [DATA_BUCK_FIELD] }
+const UPLOAD_PED_PAGE = { fields: [PROJECT_DESC_FIELD, UPLOAD_PEDIGREE_FIELD] }
+const ADD_DATA_UPLOAD_PED_PAGE = { fields: [UPLOAD_PEDIGREE_FIELD] }
+
+const FORM_WIZARD_PAGES = [GRANT_ACCESS_PAGE, VALIDATE_VCF_PAGE, UPLOAD_PED_PAGE]
+const FORM_ADD_DATA_WIZARD_PAGES = [GRANT_ACCESS_PAGE, ADD_DATA_VALIDATE_VCF_PAGE, ADD_DATA_UPLOAD_PED_PAGE]
```

Comment:
This is kind of difficult to read, I think there are a lot more constants defined than there really need to be. Can we do the following instead:

```
const GRANT_ACCESS_PAGE = { fields: [AGREE_CHECKBOX], onPageSubmit: postWorkspaceValues('grant_access') }
const VALIDATE_VCF_PAGE = { fields: [DATA_BUCK_FIELD], onPageSubmit: postWorkspaceValues('validate_vcf') },
const NEW_PROJECT_PAGES = [
  GRANT_ACCESS_PAGE,
  { ... VALIDATE_VCF_PAGE, fields: [...VALIDATE_VCF_PAGE.fields, SAMPLE_TYPE_FIELD, REQUIRED_GENOME_FIELD] },
  { fields: [PROJECT_DESC_FIELD, UPLOAD_PEDIGREE_FIELD] },
]
const ADD_DATA_PAGES = [
  GRANT_ACCESS_PAGE,
  VALIDATE_VCF_PAGE,
  { fields: [UPLOAD_PEDIGREE_FIELD] }
]
```